iceoryx_posh/test/moduletests/test_gw_base.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2021 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Please rename the file into `test_gateway_gateway_base.cpp`.

Reply:
Looks like it's done.